### DISCUSSION TITLE
Why does https://<URL to your instance>/config need to be available?

### DISCUSSION THREAD

**Discussion by andersnordh** [searxng/searxng#3855] [Q&A] [answered]
Hi, as the title say, why?

Have a good one.

//Anders

**Comment by return42** [comment, marked as answer]
Its used by check.searx.space / see https://searx.space/# and [searx-instance: "For SearXNG instance operators"](https://github.com/searxng/searx-instances?tab=readme-ov-file#for-searxng-instance-operators)
